assets/css/meta-boxes.css

```diff
@@ -8,6 +8,15 @@
 	vertical-align: middle;
 }
 
+#apple-news-publish a {
+  word-wrap: break-word;
+}
+
+#apple-news-publish button {
+  display: block !important;
```

Comment:
Can you add an inline comment explaining why `!important` is required here?

Reply:
Sure! Basically it's because we're overriding a WP style that is much more specific than this one.